scripts-dev/release.py

```diff
@@ -190,12 +190,24 @@ def run():
     # Generate changelogs
     subprocess.run("python3 -m towncrier", shell=True)
 
-    # Generate debian changelogs if its not an RC.
-    if not rc:
-        subprocess.run(
-            f'dch -M -v {new_version} "New synapse release {new_version}."', shell=True
-        )
-        subprocess.run('dch -M -r -D stable ""', shell=True)
+    # Generate debian changelogs
+    assert not parsed_new_version.is_devrelease
+    assert not parsed_new_version.is_postrelease
```

Comment:
Looks like these checks should be higher up, since we also probably don't want to go through all of the previous steps if the release isn't either an RC or a stable release.